crates/wdk-build/src/cargo_make.rs

```diff
@@ -618,10 +618,15 @@ fn get_path_from_env(
 ) -> PathBuf {
     env::var(env_var).map_or_else(
         |e| {
-            trace!(
-                "Could not read env var '{env_var}': {e:?}. Constructing path from WDK content \
-                 root"
-            );
+            const COMMON_MSG: &str = "Constructing path from WDK content root instead";
```

Comment:
nit: more descriptive name for this constant

Reply:
Renamed it to `FALLBACK_MSG`